Packs/Phishing/ReleaseNotes/3_5_8.md

```diff
@@ -0,0 +1,15 @@
+
+#### Layouts
+
+##### Phishing Incident v3
+
+- Added "Selected indicators to block" section and incident field. this section will show the indicators chosen to be blocked during the investigation.
+
+#### Playbooks
+
+##### Phishing - Generic v3
+
+- "Block Indicators - Generic v2" sub-playbook replaced with "Block Indicators - generic v3" and the conditional task "Should indicators be blocked automatically?" has been removed. Block indicators method will be determined in the sub-playbook itself using the "AutoBlockIndicators" input.
+- "BlockIndicators" input will now determine whether to automatically block indicators or selecting manually which indicators to block, if any.
```

Comment:
Add the full description that you have edited for this input. will make sense to the users to review that once they updated the pack.